jwql/instrument_monitors/nirspec_monitors/ta_monitors/wata_monitor.py

```diff
@@ -694,31 +708,13 @@ def get_uncal_names(self, file_list):
         """
         good_files = []
         for filename in file_list:
-            # Names look like: jw01133003001_02101_00001_nrs2_cal.fits
-            if '_uncal' not in filename:
-                suffix2replace = filename.split('_')[-1]
-                filename = filename.replace(suffix2replace, 'uncal.fits')
-            if filename not in good_files:
-                good_files.append(filename)
-        return good_files
-
-    def get_uncal_names(self, file_list):
-        """Replace the last suffix for _uncal and return list.
-        Parameters
-        ----------
-        file_list : list
-            List of fits files
-        Returns
-        -------
-        good_files : list
-            Filtered list of uncal file names
-        """
-        good_files = []
-        for filename in file_list:
-            # Names look like: jw01133003001_02101_00001_nrs2_cal.fits
-            if '_uncal' not in filename:
+            if filename.endswith('.fits'):
```

Comment:
I think this shouldn't have any impact on the amount of input data.  The `endswith('.fits')` clause is for processing results from a MAST search; the other one is for processing results from a model query.  They should return identical results, but the model query is generally faster. 

I left both methods in place for now because they both have potential failure points.  The MAST connection is sometimes unreliable.  The model query is sometimes incomplete, depending on how well synced our models are with MAST.  I'd like to see how it goes with the model query for now, but we might want to revert to MAST later.

Reply:
That makes sense. I saw the comparison of the two datasets and agree the faster the better but having a backup plan is always a good idea. Thanks!